data/role/jenkins-dev.yaml

```diff
@@ -9,6 +9,13 @@ psick::jenkins::jcasc::options_hash:
     Name: 'jenkins-slave-windows-dev'
     Role: 'jenkins-slave'
     Env: 'dev'
+  linux_slave_ami: 'ami-f61ab08c'
+  linux_slave_securityGroups: 'ssh-access, all-egress-dev, vpn-ingress-dev'
```

Comment:
Please don't use camelCase, i.e. `linux_slave_security_groups`. I do see that it was done like that above for windows. If it is a big problem to change it, then don't bother.